src/app/modules/form/admin-form/__tests__/admin-form.service.spec.ts

```diff
@@ -2628,4 +2630,124 @@ describe('admin-form.service', () => {
       expect(twilioCacheSpy).toHaveBeenCalledWith(MSG_SRVC_NAME)
     })
   })
+
+  describe('updatePayments', () => {
+    // Arrange
+    const mockFormId = new ObjectId().toString()
+    const updatedPaymentSettings = {
+      enabled: true,
+      target_account_id: 'someId',
+      publishable_key: 'somekey',
+      amount_cents: 5000,
+      description: 'some description',
+    } as PaymentsUpdateDto
+
+    const mockUpdatedForm = {
+      _id: mockFormId,
+      payments: updatedPaymentSettings,
+    }
+
+    it('should return InvalidPaymentAmountError if payment amount exceeds maxPaymentAmountCents', async () => {
+      // Arrange
+
+      const updatedPaymentSettingsExceeded = {
+        enabled: true,
+        target_account_id: 'someId',
+        publishable_key: 'somekey',
+        amount_cents: 500000,
+        description: 'some description',
+      } as PaymentsUpdateDto
```

Comment:
Should this definition make use of the spread operator to ensure that other params are the same as the ones in `updatedPaymentSettings`? So there'll be less tech debt in the future if we change our API payloads. So like:
```suggestion
      const updatedPaymentSettingsExceeded = {
        ...updatedPaymentSettings,
        amount_cents: 500000,
      } as PaymentsUpdateDto
```

Reply:
Sounds good! made the change in [fa5308e](https://github.com/opengovsg/FormSG/pull/5972/commits/fa5308ee0cf1836d0c9ac2ea43ecfb259f8453a1)